crates/frontend/src/pages/dimensions.rs

```diff
@@ -22,6 +23,8 @@ pub struct RowData {
 #[component]
 pub fn dimensions() -> impl IntoView {
     let tenant_rs = use_context::<ReadSignal<String>>().unwrap();
+    let (delete_modal_visible, set_delete_modal_visible) = create_signal(false);
```

Comment:
can be optimised to one single signal, but DeleteDimensions modal bad structure seem to be a blocker for these changes

Reply:
Agree